index.js

```diff
@@ -1,42 +1,53 @@
 module.exports = {
   root: true,
   extends: [
-    'eslint:recommended',
-    'plugin:@typescript-eslint/eslint-recommended',
-    'plugin:@typescript-eslint/recommended',
+    "eslint:recommended",
```

Comment:
why the change to double quote?  doesn't that conflict with our linter (or prettier) rules?

Reply:
It used the default prettier config, I've updated it to use our own.